_drafts/2015-10-02-microcoap-and-ff-copper.md

```diff
@@ -0,0 +1,245 @@
+---
+layout: post
+title: CoAP on RIOT
+subtitle: implementing and testing with microcoap and copper
+categories: CoAP
+author: lotte
+date: 2015-03-17
+---
+
+In [one of our last posts](http://watr.li/what-is-coap.html), we explained how the [Constrained Application Protocol](http://coap.technology) enables us to exchange data between nodes in the Internet of Plants using a request/reply cycle similar to that of HTTP. 
+To do this, we need both of our node types– plant nodes and display nodes– to be able to send, process and answer CoAP requests.
+
+This post explains how to implement a simple CoAP server on our *plant nodes*, which run [RIOT](http://www.riot-os.org), using RIOTs [microcoap](https://github.com/1248/microcoap) package. 
+
+Since no special modifications to the code are needed to get it to run on RIOT, this guide may also be useful to you if you're looking to run microcoap on Linux or your Arduino.{: .alert .alert-info }
```

Comment:
das alert muss glaub ich in die zeile nach dem absatz. testest du auch mit jekyll?